invenio_oauthclient/templates/invenio_oauthclient/_macros.html

```diff
@@ -18,22 +18,6 @@
 {% endmacro %}
 
 
-{% macro form_errors(form) %}
```

Comment:
This macro was not used anywhere else except invenio-accounts and invenio-oauthclient (see [organization search](https://github.com/search?p=3&q=org%3Ainveniosoftware+form_errors&type=Code). However, both modules also include the code for it in them, so there is no need to have it here. The same applies to the semanti-ui macro.